packages/atlas/src/joystream-lib/extrinsics.ts

```diff
@@ -72,6 +72,7 @@ type PublicExtrinsic<TxFunction, ReturnValue> = TxFunction extends (...a: infer
   : never
 
 const PERMILLS_PER_PERCENTAGE = 10
+const PERQUINTILLS_PER_PERCENTAGE = 100
```

Comment:
`Perquintill` are ["parts per Quintillion"](https://docs.rs/sp-runtime/latest/sp_runtime/struct.Perquintill.html) so I think would be:
```ts
const PERQUINTILLS_PER_PERCENTAGE = 10**16
```
($`10^{18 - 2}`$) but  the `PERQUINTILLS_PER_PERCENTAGE` would have to be a `BN` because $`10^{16}`$ is more than `Number.MAX_SAFE_INTEGER` so if `PERQUINTILLS_PER_PERCENTAGE` is a `number`, `BN` will throw an exception for all ratio that aren't extremely low.